source/plugins/go/src/telemetry.go

```diff
@@ -360,40 +365,27 @@ func InitializeTelemetryClient(agentVersion string) (int, error) {
 	CommonProperties = make(map[string]string)
 	CommonProperties["Computer"] = Computer
 	CommonProperties["WorkspaceID"] = WorkspaceID
-	CommonProperties["ControllerType"] = os.Getenv("CONTROLLER_TYPE")
-	CommonProperties["AgentVersion"] = agentVersion
+	CommonProperties["ControllerType"] = ControllerType[strings.ToLower(os.Getenv("CONTROLLER_TYPE"))]
```

Comment:
validate these changes. and also, we would need to update our dashboard once after the change rollout. please add a task for this.

Reply:
![image](https://github.com/microsoft/Docker-Provider/assets/18661789/b6c87069-56d2-45d5-a418-3687c60d182d)
![image](https://github.com/microsoft/Docker-Provider/assets/18661789/e2d66731-5ba8-40ec-8192-0443a31cd014)
![image](https://github.com/microsoft/Docker-Provider/assets/18661789/ed4d0dc3-6ee9-4b2d-b2a1-0343133ae801)

Reply:
Tested on both windows and all tables